src/mono/wasm/runtime/driver.c

```diff
@@ -505,6 +503,12 @@ mono_wasm_load_runtime (const char *unused, int debug_level)
 #else
 	mono_jit_set_aot_mode (MONO_AOT_MODE_INTERP_ONLY);
 
+#ifdef ENABLE_METADATA_UPDATE
+	if (monoeg_g_hasenv ("MONO_METADATA_UPDATE")) {
```

Comment:
Shouldn't this be some kind of option ?

Reply:
Made an issue to figure out the official way to do this #47274​​​​​​​